doc/devel/coding_guide.rst

```diff
@@ -13,8 +13,90 @@
 Pull request guidelines
 ***********************
 
-Pull requests (PRs) are the mechanism for contributing to Matplotlibs code and
-documentation.
+`Pull requests (PRs) on GitHub <https://docs.github.com/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests>`__
```

Comment:
Note, you don't need to keep everything in the backticks on one line; you can break on the spaces.